### ISSUE TITLE
Exception from frozen_world preset

### ISSUE THREAD

**Issue by Browsit** [PolyhedralDev/TerraOverworldConfig#136]
This server is running Paper version git-Paper-58 (MC: 1.19) (Implementing API version 1.19-R0.1-SNAPSHOT) (Git: e294802)
Terra-bukkit-6.2.0-BETA+7d056bd88-shaded using Bukkit method (no Multiverse)

Using default pack with only change being pack.yml modified with
`biomes: $biome-providers/presets/frozen_world.yml:biomes`

<details>
  <summary>Startup exception</summary>
  
```
[07:37:01] [Server thread/INFO]: [com.dfsek.terra.registry.master.ConfigRegistry] Loading ZIP archive: default.zip
[07:37:02] [Server thread/INFO]: [com.dfsek.terra.config.pack.ConfigPackImpl] Loading config pack "OVERWORLD:OVERWORLD"
[07:37:05] [Server thread/INFO]: [com.dfsek.terra.config.pack.ConfigPackImpl] Loaded config pack "OVERWORLD:OVERWORLD" v1.2.0 by Astrash, Sancires, Aureus in 4270.0032ms.
[07:37:05] [Server thread/ERROR]: [com.dfsek.terra.registry.master.ConfigRegistry] Error loading config pack default.zip
com.dfsek.tectonic.api.exception.LoadException: Failed to load configuration:

	Configuration: pack.yml
	Message: Unexpected exception thrown during type loading: Biome Pipeline leaks ephemeral biome "mountains-small-polar". Ensure there is a stage to guarantee replacement of the ephemeral biome. Biomes: 
    - ARID_SPIKES:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - BADLANDS_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - BADLANDS_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - CHAPARRAL:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - CHAPARRAL_FLATS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - CRACKED_BADLANDS_PLATEAU:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DARK_FOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DARK_FOREST_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_ROCKY_BUMPY_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_WHITE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_WHITE_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_WILD_HIGHLANDS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - EUCALYPTUS_FOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - EVERGREEN_FOREST_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FOREST_FLATS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_ARCHIPELAGO:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_BEACH:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_MARSH:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_WETLANDS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - GRASS_SAVANNA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - GRASS_SAVANNA_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - GRASS_SAVANNA_LOW_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - JUNGLE_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - JUNGLE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - LARGE_MONSOON_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - LOW_CHAPARRAL:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MANGROVE_SWAMP:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MARSH:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MONSOON_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MOORLAND:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - OAK_SAVANNA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - OVERGROWN_CLIFFS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - RAINFOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - RAINFOREST_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - ROCKY_BUMPY_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA_LOW_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA_OVERHANGS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SHRUBLAND:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_ERODED_TERRACED_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_ERODED_TERRACED_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_TERRACED_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_TERRACED_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - STEPPE:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SWAMP:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TAIGA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TAIGA_FLATS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TAIGA_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TEMPERATE_ALPHA_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TEMPERATE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TEMPERATE_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - WILD_HIGHLANDS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - WOODED_BUTTES:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_LOW_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_PLAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - badlands-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - birch-flats-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - birch-forest-hills-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - birch-forest-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-hills-large:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-hills-small:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-mountains-large:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-mountains-small:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - deep-ocean-border:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - deep-ocean-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - desert-flats-microvariants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - desert-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - evergreen-flats-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - evergreen-forest-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - flat-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - forest-hills-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - hills-large-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - hills-small-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - jungle-flats-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - jungle-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - mountains-large-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - mountains-small-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - ocean-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - plains-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - temperate-mountains-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - wild-mountains-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate

	Path: biomes
	Full Path: 
		From configuration "pack.yml"
		In entry "biomes"
		From configuration "biome-providers/presets/frozen_world.yml"

	at com.dfsek.tectonic.api.loader.ConfigLoader.getObject(ConfigLoader.java:222) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.tectonic.api.loader.ConfigLoader.loadType(ConfigLoader.java:204) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.tectonic.api.loader.ConfigLoader.loadValue(ConfigLoader.java:161) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.tectonic.impl.loading.template.ReflectiveTemplateLoader.load(ReflectiveTemplateLoader.java:42) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.tectonic.api.loader.ConfigLoader.load(ConfigLoader.java:151) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.tectonic.api.loader.ConfigLoader.load(ConfigLoader.java:147) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.config.pack.ConfigPackImpl.<init>(ConfigPackImpl.java:233) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.config.pack.ConfigPackImpl.<init>(ConfigPackImpl.java:135) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.registry.master.ConfigRegistry.load(ConfigRegistry.java:77) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.registry.master.ConfigRegistry.loadAll(ConfigRegistry.java:67) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.AbstractPlatform.lambda$load$0(AbstractPlatform.java:152) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.event.EventContextImpl.lambda$handle$0(EventContextImpl.java:50) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at com.dfsek.terra.event.EventContextImpl.handle(EventContextImpl.java:50) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.event.FunctionalEventHandlerImpl.lambda$handle$0(FunctionalEventHandlerImpl.java:55) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at java.util.ArrayList.forEach(ArrayList.java:1511) ~[?:?]
	at com.dfsek.terra.event.FunctionalEventHandlerImpl.handle(FunctionalEventHandlerImpl.java:48) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.event.EventManagerImpl.lambda$callEvent$0(EventManagerImpl.java:38) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at java.util.HashMap$Values.forEach(HashMap.java:1065) ~[?:?]
	at com.dfsek.terra.event.EventManagerImpl.callEvent(EventManagerImpl.java:38) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.terra.bukkit.TerraBukkitPlugin.onEnable(TerraBukkitPlugin.java:59) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:370) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:536) ~[paper-api-1.19-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugin(CraftServer.java:563) ~[paper-1.19.jar:git-Paper-58]
	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.enablePlugins(CraftServer.java:477) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:279) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1121) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302) ~[paper-1.19.jar:git-Paper-58]
	at java.lang.Thread.run(Thread.java:833) ~[?:?]
Caused by: java.lang.IllegalArgumentException: Biome Pipeline leaks ephemeral biome "mountains-small-polar". Ensure there is a stage to guarantee replacement of the ephemeral biome. Biomes: 
    - ARID_SPIKES:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - BADLANDS_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - BADLANDS_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - CHAPARRAL:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - CHAPARRAL_FLATS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - CRACKED_BADLANDS_PLATEAU:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DARK_FOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DARK_FOREST_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_ROCKY_BUMPY_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_WHITE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_TEMPERATE_WHITE_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - DRY_WILD_HIGHLANDS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - EUCALYPTUS_FOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - EVERGREEN_FOREST_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FOREST_FLATS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_ARCHIPELAGO:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_BEACH:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_MARSH:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - FROZEN_WETLANDS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - GRASS_SAVANNA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - GRASS_SAVANNA_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - GRASS_SAVANNA_LOW_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - JUNGLE_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - JUNGLE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - LARGE_MONSOON_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - LOW_CHAPARRAL:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MANGROVE_SWAMP:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MARSH:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MONSOON_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MOORLAND:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - OAK_SAVANNA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - OVERGROWN_CLIFFS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - RAINFOREST:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - RAINFOREST_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - ROCKY_BUMPY_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA_LOW_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SAVANNA_OVERHANGS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SHRUBLAND:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_ERODED_TERRACED_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_ERODED_TERRACED_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_TERRACED_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SNOWY_TERRACED_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - STEPPE:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - SWAMP:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TAIGA:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TAIGA_FLATS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TAIGA_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TEMPERATE_ALPHA_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TEMPERATE_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - TEMPERATE_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - WILD_HIGHLANDS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - WOODED_BUTTES:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_LOW_HILLS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_MOUNTAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_MOUNTAINS_RIVER:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - XERIC_PLAINS:com.dfsek.terra.addons.biome.pipeline.api.delegate.DelegatedBiome
    - badlands-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - birch-flats-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - birch-forest-hills-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - birch-forest-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-hills-large:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-hills-small:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-mountains-large:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - coast-mountains-small:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - deep-ocean-border:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - deep-ocean-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - desert-flats-microvariants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - desert-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - evergreen-flats-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - evergreen-forest-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - flat-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - forest-hills-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - hills-large-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - hills-small-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - jungle-flats-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - jungle-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - mountains-large-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - mountains-small-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - ocean-polar:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - plains-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - temperate-mountains-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate
    - wild-mountains-variants:com.dfsek.terra.addons.biome.pipeline.api.delegate.EphemeralBiomeDelegate

	at com.dfsek.terra.addons.biome.pipeline.BiomePipelineProvider.lambda$new$2(BiomePipelineProvider.java:68) ~[?:?]
	at java.lang.Iterable.forEach(Iterable.java:75) ~[?:?]
	at com.dfsek.terra.addons.biome.pipeline.BiomePipelineProvider.<init>(BiomePipelineProvider.java:56) ~[?:?]
	at com.dfsek.terra.addons.biome.pipeline.config.BiomePipelineTemplate.get(BiomePipelineTemplate.java:52) ~[?:?]
	at com.dfsek.terra.addons.biome.pipeline.config.BiomePipelineTemplate.get(BiomePipelineTemplate.java:24) ~[?:?]
	at com.dfsek.terra.config.loaders.GenericTemplateSupplierLoader.load(GenericTemplateSupplierLoader.java:50) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at com.dfsek.tectonic.api.loader.ConfigLoader.getObject(ConfigLoader.java:210) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	... 29 more
[07:37:05] [Server thread/INFO]: [com.dfsek.terra.AbstractPlatform] Loaded packs.
[07:37:05] [Server thread/INFO]: [com.dfsek.terra.bukkit.nms.v1_19_R1.AwfulBukkitHacks] Hacking biome registry...
[07:37:06] [Server thread/INFO]: [com.dfsek.terra.bukkit.nms.v1_19_R1.AwfulBukkitHacks] Doing tag garbage....
[07:37:06] [Server thread/WARN]: **** SERVER IS RUNNING IN OFFLINE/INSECURE MODE!
[07:37:06] [Server thread/WARN]: The server will make no attempt to authenticate usernames. Beware.
[07:37:06] [Server thread/WARN]: While this makes the game possible to play without internet access, it also opens up the ability for hackers to connect with any username they choose.
[07:37:06] [Server thread/WARN]: To change this, set "online-mode" to "true" in the server.properties file.
[07:37:06] [Server thread/INFO]: Preparing level "world"
[07:37:06] [Server thread/ERROR]: [Terra] Could not set generator for default world 'world': Plugin 'Terra v6.2.0-BETA+7d056bd88
java.lang.IllegalArgumentException: No such config pack "OVERWORLD"
	at com.dfsek.terra.bukkit.TerraBukkitPlugin.lambda$getDefaultWorldGenerator$1(TerraBukkitPlugin.java:176) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at java.util.Optional.orElseThrow(Optional.java:403) ~[?:?]
	at com.dfsek.terra.bukkit.TerraBukkitPlugin.lambda$getDefaultWorldGenerator$2(TerraBukkitPlugin.java:175) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at java.util.HashMap.computeIfAbsent(HashMap.java:1220) ~[?:?]
	at com.dfsek.terra.bukkit.TerraBukkitPlugin.getDefaultWorldGenerator(TerraBukkitPlugin.java:174) ~[Terra-bukkit-6.2.0-BETA+7d056bd88-shaded.jar:?]
	at org.bukkit.craftbukkit.v1_19_R1.CraftServer.getGenerator(CraftServer.java:1642) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:525) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:419) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:306) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1121) ~[paper-1.19.jar:git-Paper-58]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:302) ~[paper-1.19.jar:git-Paper-58]
```
  
</details>

**Comment by Astrashh** [comment]
The frozen world preset was removed in place of stage presets but was marked as `TODO` before removal, your version of the overworld probably came before this commit: https://github.com/PolyhedralDev/TerraOverworldConfig/commit/c583d4045a012ac8f75622492959a3c7e2edadb9

**Comment by Browsit** [comment]
@Astrashh Our version of the overworld is the one included with 6.2.0-BETA on SpigotMC. We were simply able to deduce that the preset existed based on the file

https://github.com/PolyhedralDev/TerraOverworldConfig/blob/master/biome-providers/presets/frozen_world.yml

If the world preset is no longer supported, it probably ought to be removed. Otherwise, we're still interested in a fix.

**Comment by Astrashh** [comment]
You can use the polar temperature stage preset as a replacement for the frozen world preset https://github.com/PolyhedralDev/TerraOverworldConfig/blob/1a87aed3e30ab021071332a0e77e2355dfde5ce2/biome-providers/presets/default.yml#L89
